### PR TITLE
All Updates so far

### PR DESCRIPTION
**Pull request by FominSergiy**
Working in a dev brunch on this up.

Progress made so far
- Got base components for the board, squares
- Set up reducers to run with thunks
- Attached moving to buttons and arrow buttons on keyboard

Left To Do:
- set up timer for 60 seconds
- overall game styling
- update my file structure (just looks bad)

### CODE REVIEW

**Comment by FominSergiy** [comment]
Left To Do:

overall game styling
update my file structure (just looks bad)

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-  "name": "tanks_game",
+  "homepage": "http://FominSergiy.github.io/dodgeFairy",
+  "name": "dodge_fairy",
```
Comment (karlowski) [inline package.json]: from tanks to fairies! :D

File: src/features/game-over/gameOver.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { useDispatch, useSelector } from "react-redux";
+import React from 'react';
+
+export const GameOver = (props) => {
```
Comment (karlowski) [inline src/features/game-over/gameOver.js:4]: looks very nice when even *game over stuff* separated in different module

File: src/utils/utils.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+    if (score >= 0) msg='Try harder next time!';
+    if (score > 5) msg='Are you sure you are using keyboard?';
+    if (score > 10) msg='Your rocket crashed back on Earth!';
+    if (score > 20) msg='I can see DODG high in the sky!';
+    if (score > 50) msg='You are really good at this!';
+    if (score > 75) msg='TO THE MOOOON!';
+
+    return msg;
+}
```
Comment (karlowski) [inline src/utils/utils.js:165]: what  is separated utils file for btw?
kinda constants but for more complex stuff like functions..?

**Review by karlowski — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (FominSergiy) [reply package.json]: right ahaha

File: src/utils/utils.js (hunk 3 above)
Comment (FominSergiy) [reply src/utils/utils.js:165]: I don't even know. I created it initially to hold all the utills for the project, but then it made sense, for some pieces, to hold utils in the same place as the code for the component.. I really need to re-think this

**Pull request merged**